doc/rst/source/supplements/potential/gravprisms.rst

```diff
@@ -76,7 +76,7 @@ Required Arguments
 *table*
     The file describing the prisms with record format *x y z_lo z_hi* [ *dx dy* ] [ *rho* ],
     where the optional items are controlled by options **-E** and **-D**, respectively.
-    Density contrasts can be given in kg/m^3 of g/cm^3. **Note**: If **-C** is used then
+    Density contrasts can be given in :math:`\mbox{kg/m}^3` of :math:`\mbox{g/cm}^3`. **Note**: If **-C** is used then
```

Comment:
Should it be 
```
:math:`\mbox{kg/m^3}`
```
?

Reply:
I think mbox is just for plain text in the middle of math, so no?

Reply:
> I think mbox is just for plain text in the middle of math, so no?

You're right. The following expressions:
```
:math:`\mbox{kg/m}^3`

:math:`\mbox{kg/m^3}`

:math:`kg/m^3`
```
are rendered like:
<img width="94" alt="image" src="https://user-images.githubusercontent.com/3974108/167261256-e7a64198-3e5f-4131-a98d-f73ec86cc019.png">


```
:math:`\mbox{kg/m}^3`
```
just gives me the wrong impression of (kg/m)^3, not kg/(m^3).

Reply:
OK, so keep as is then.